### PR TITLE
Moved the get_component_init_parameters to class_utils

### PR DESCRIPTION
**Pull request by yhwen**
Fixes # .

### Description

Moved the get_component_init_parameters to class_utils for sharing function.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/fuel/utils/class_utils.py [hunk 1]
```diff
@@ -136,3 +136,20 @@ def get_module_name(self, class_name) -> Optional[str]:
...
+    constructor = class__.__init__
+    constructor__parameters = inspect.signature(constructor).parameters
+    parameters.update(constructor__parameters)
+    if "args" in constructor__parameters.keys() and "kwargs" in constructor__parameters.keys():
+        for item in class__.__bases__:
+            parameters.update(_retrieve_parameters(item, parameters))
+    return parameters
+
+
+def get_component_init_parameters(component):
```
Comment (yanchengnv) [inline nvflare/fuel/utils/class_utils.py:151]: Please add unit test cases for simple and complex (subclassing) object structures.

**Review by yanchengnv — CHANGES_REQUESTED** [review]
Looks good. Please add docstrings and unit test cases for get_component_init_parameters.

File: nvflare/fuel/utils/class_utils.py (hunk 1 above)
Comment (yhwen) [reply nvflare/fuel/utils/class_utils.py:151]: Added docstring and unit test.

**Review by holgerroth — APPROVED** [review]
Very useful!

**Review by yanchengnv — APPROVED** [review]
LGTM

**Pull request merged**